tags/terminal/unix_utilities.py

```diff
@@ -83,3 +85,22 @@
 
 mod.list("unix_utility", desc="A common utility command")
 ctx.lists["self.unix_utility"] = unix_utilities
+
+# 2. arguments
+
+default_unix_arguments = {"--help": "help"}
```

Comment:
This needs to be swapped. 
```suggestion
default_unix_arguments = {"help": "--help"}
```
Unintuitively, `get_list_from_csv` swaps the order of the columns.

Reply:
I took all the commands currently within my `unix_utilities.csv` and looked at the output of their `--help` option:
```
cat unix_utilities.csv | grep  -v command | cut -f1 -d, | cut -f1 -d' ' | sort -u | xargs -I% bash -c "% --help || true | sort -u" | grep -oE -- '--[a-zA-Z_-]+' | sort | uniq -c | sort -n
```
I think this is going to give a good starting point of often-used parameters for the initial `default_unix_arguments`.
For example:
```
--all,all
--auth,auth
--backup,backup
--binary,binary
--bytes,bytes
--check,check
--context,context
--debug,debug
--file,file
--files,files
--filter,filter
--follow,follow
--force,force
--form,form
--help,help
--ignore-case,ignore-case
--include,include
--json,json
--lines,lines
--null,null
--output,output
--pretty,pretty
--print,print
--quiet,quiet
--raw,raw
--recursive,recursive
--relative,relative
--reverse,reverse
--silent,silent
--tag,tag
--terse,terse
--text,text
--timeout,timeout
--times,times
--time,time
--user,user
--verbose,verbose
--verify,verify
--version,version
--zero-terminated,zero-terminated
--zero,zero
```